foxtrot-core/src/main/java/com/flipkart/foxtrot/core/datastore/DataStore.java

```diff
@@ -25,9 +26,15 @@
  * Date: 12/03/14
  * Time: 9:17 PM
  */
-public interface DataStore {
+public interface DataStore extends Identifiable {
```

Comment:
Inherting this interface is not required as the getId function is not used anywhere.